cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -272,7 +268,26 @@ data TxOut era
   = TxOut (AddressInEra era) (TxOutValue era)
   deriving Generic
 
-deriving instance IsCardanoEra era => ToJSON (TxOut era)
+instance IsCardanoEra era => ToJSON (TxOut era) where
+  toJSON (TxOut (AddressInEra addrType addr) val) =
+    case addrType of
+      ByronAddressInAnyEra ->
+        let hexAddr = Text.decodeUtf8 $ serialiseToRawBytesHex addr
+        in object [ hexAddr .= toJSON val ]
+      ShelleyAddressInEra sbe ->
+        case sbe of
+          ShelleyBasedEraShelley ->
+            let hexAddr = Text.decodeUtf8 $ serialiseToRawBytesHex addr
+            in object [ hexAddr .= toJSON val ]
+          ShelleyBasedEraAllegra ->
+            let hexAddr = Text.decodeUtf8 $ serialiseToRawBytesHex addr
+            in object [ hexAddr .= toJSON val ]
+          ShelleyBasedEraMary ->
+            let hexAddr = Text.decodeUtf8 $ serialiseToRawBytesHex addr
+            in object [ hexAddr .= toJSON val ]
```

Comment:
This is a bit weird. Why not just use two fields "address" and "value"? Why have this field name that is the address?

Reply:
So currently the JSON output looks like this: https://pastebin.com/tV2Ce0Qk

I did it that way to keep it more concise but I can change it to have two fields ("address" and "value")

Reply:
I also prefer `address` and `value`.